File: algos/isPalindrome/isPalindrome.ts
var isPalindrome = function (s) {
  const anmc = 'abcdefghijklmnopqrstuvwxyz0123456789';
  const alteredString = s.replace(/[^a-zA-Z0-9]/g, "");
  return alteredString.toLowerCase() === alteredString.split('').reverse().join('').toLowerCase();
};

File: algos/isPalindrome/isPalindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        # use isalnum for is alphanumeric
        alpha = ''.join(e for e in s if e.isalnum()).lower()

        reverse = alpha[::-1]
        if alpha == reverse:
            return True
        else:
            return False